Sources/Transmute/transmute().swift

```diff
@@ -66,18 +66,10 @@ public func transmute(packages: [Package], rootdir: String) throws -> ([Module],
     // ensure modules depend on the modules of any dependent packages
     fillModuleGraph(packages, modulesForPackage: { map[$0]! })
 
-    var set = Set<Module>()
-    var stack = packages.flatMap{ map[$0] ?? [] }
-    var modules = [Module]()
+    let depPackages = packages.filter{ $0.path != rootdir }
```

Comment:
Agreed. Refactored `get()` to return source package and dependency packages separately.